ui/js/graphs.js

```diff
@@ -797,6 +804,71 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
         });
     };
 
+    var loadingExtraDataPromise = $q.defer();
+    var addRelatedPlatforms = function(relatedSeries) {
+        PhSeries.getAllSeries(relatedSeries.projectName,
+            $scope.timeRange, optionCollectionMap).then(
```

Comment:
Maybe we could use the same method we applied in bug 1189519(https://github.com/mozilla/treeherder/pull/914/files#diff-5266f838afcedb1798e0cf4a88cba462R889)? @wlach